synapse/Dockerfile

```diff
@@ -6,5 +6,10 @@ RUN SYNAPSE_VERSION="0.19.2" && BUILD_PKGS="redhat-rpm-config libtiff-devel libj
  && dnf remove -y $BUILD_PKGS \
  && rm -rf /root/.cache \
  && rm -rf /var/cache/dnf/*
+RUN useradd docker
```

Comment:
Can this be a system account, and be set to a fixed UID? (I've seen 797 used in [other Dockerfiles](https://hub.docker.com/r/wernight/plex-media-server/~/dockerfile/).)

Main thing is I worry about some process changing how UIDs are allocated and everything breaks because I need to manually chown the /data volume.